recipes/vulkan-validationlayers/all/conandata.yml

```diff
@@ -2,6 +2,9 @@ sources:
   "1.2.182":
     url: "https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/v1.2.182.tar.gz"
     sha256: "5a1f7027c06a8e5ae777d9053b5ce46f10ca623806a43332eb2da06fe46476d4"
+  "1.2.172":
+    url: "https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/v1.2.172.tar.gz"
+    sha256: "0e29bf2545658177d37052bcce09f29d3f464caf31d00a84c37368dd1a660d29"
```

Comment:
does it need the same patch than 1.2.182?

Reply:
It worked for my purposes just out-of-the-box

Reply:
Well, it doesn't answer to my question. But looking at patches comments and related upstream PRs, 1.2.172 doesn't need patches.